modules/tinymce/CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The default value for `end_container_on_empty_block` option has been changed to `'blockquote'` #TINY-6559
 - Custom elements are now treated as non-empty elements via the schema #TINY-4784
 - The autocompleter menu element is now positioned instead of the wrapper #TINY-6476
+- `mceLink` command can now take `true` value to always open link dialog #TINY-8057
```

Comment:
We possibly don't really want to document this, given it's largely internal. If we do, then this is an improvement so should be in the `Improved` section instead.

Reply:
Seeing as the documentation needs to be updated I was considering including how to use the `mceLink` command to open the dialog even with quicklink enabled as that may be important, so I'll ask documentation how best to describe this for the changelog also